src/zenml/utils/pipeline_docker_image_builder.py

```diff
@@ -465,8 +482,106 @@ def _gather_requirements_files(
                     ", ".join(f"`{r}`" for r in integration_requirements_list),
                 )
 
+        # Generate requirements files for all ZenML Hub plugins
+        if docker_settings.required_hub_plugins:
+            (
+                hub_packages,
```

Comment:
This code was really confusing to me because I couldn't really get what `hub_packages` and `hub_requirements` refer to until I finished reading all the code. Are there some more intuitive names we could use, e.g. `hub_requirements` and `pypi_requirements` or something along those lines?

Reply:
I renamed them to `hub_pypi_requirements` and `hub_internal_requirements`, is that more clear?